src/main/java/com/snake/game/Window.java

```diff
@@ -40,9 +40,13 @@ public Window(){
         }
 
         // passing this value to the controller
-        ThreadsController c = new ThreadsController(new Tuple(5,5), new Tuple(15,15));
-        //Let's start the game now..
-        c.start();
+        if (gamemode == "two"){
+            ThreadsController c = new ThreadsController(new Tuple(5,5), new Tuple(15,15));
+            c.start();
+        }else{
+            ThreadsController c = new ThreadsController(new Tuple(5,5));
+            c.start();
+        }
```

Comment:
```suggestion
        ThreadsController c
        if (gamemode == "two"){
            c = new ThreadsController(new Tuple(5,5), new Tuple(15,15));
        }else{
            c = new ThreadsController(new Tuple(5,5));
        }
        c.start();
```
This just eliminates a bit of code repetition.

Reply:
Maybe it wasn't clear in my comment but the game is working fine now for both game modes. And sounds good, thanks for the changes.